plugins/inputs/intel_powerstat/README.md

```diff
@@ -48,13 +48,50 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## by the plugin.
   ## Supported options:
   ##   "cpu_frequency", "cpu_c0_state_residency", "cpu_c1_state_residency",
-  ##   "cpu_c6_state_residency", "cpu_busy_cycles", "cpu_temperature",
-  ##   "cpu_busy_frequency"
-  ## ATTENTION: cpu_busy_cycles is DEPRECATED - use cpu_c0_state_residency
+  ##   "cpu_c3_state_residency", "cpu_c6_state_residency", "cpu_c7_state_residency",
+  ##   "cpu_busy_cycles", "cpu_temperature", "cpu_busy_frequency",
+  ##   "cpu_c0_substate_c01", "cpu_c0_substate_c02", "cpu_c0_substate_c0_wait"
+  ## ATTENTION: cpu_busy_cycles is DEPRECATED - use cpu_c0_state_residency instead.
   # cpu_metrics = []
+
+  ## Optionally the user can choose for which CPUs metrics configured in cpu_metrics array should be gathered.
+  ## Can't be combined with excluded_cpus.
+  ## Empty or missing array means CPU metrics are gathered for all CPUs.
+  ## e.g. ["0-3", "4,5,6"] or ["1-3,4"]
+  # included_cpus = []
+
+  ## Optionally the user can choose which CPUs should be excluded from gathering metrics configured in cpu_metrics array.
+  ## Can't be combined with included_cpus.
+  ## Empty or missing array means CPU metrics are gathered for all CPUs.
+  ## e.g. ["0-3", "4,5,6"] or ["1-3,4"]
+  # excluded_cpus = []
+
+  ## Filesystem location of JSON file that contains PMU event definitions.
+  ## Mandatory only for perf-related metrics (cpu_c0_substate_c01, cpu_c0_substate_c02, cpu_c0_substate_c0_wait).
+  # event_definitions = ""
+
+  ## The user can set the timeout duration for MSR reading.
+  ## Enabling this timeout can be useful in situations where, on heavily loaded systems,
+  ## the code waits too long for a kernel response to MSR read requests.
+  ## 0 disables the timeout (default).
+  # msr_read_timeout = 0
 ```
 
-## Example: Configuration with no per-CPU telemetry
+### Configuration notes
+
+1. The configuration of `included_cpus` or `excluded_cpus` may affect the ability to collect `package_metrics`.
+   Some of them (`max_turbo_frequency`, `cpu_base_frequency`, and `uncore_frequency`) need to read data
+   from exactly one processor for each package. If `included_cpus` or `excluded_cpus` exclude all processors
+   from the package, reading the mentioned metrics for that package will not be possible.
+2. `event_definitions` JSON file for specific architecture can be found at [perfmon](https://github.com/intel/perfmon).
+   A script to download the event definition that is appropriate for current environment (`event_download.py`) is
+   available at [pmu-tools](https://github.com/andikleen/pmu-tools).
+   For perf-related metrics supported by this plugin, an event definition JSON file
+   with events for the `core` is required.
+
+   For example: `sapphirerapids_core.json` or `GenuineIntel-6-8F-core.json`.
```

Comment:
This file is a test file. The file with valid perf events is located at [`perfmon`](https://github.com/intel/perfmon) repo. I would not link it to this particular file.

Reply:
@Hipska Yes, as @ggalieroc wrote: event files should be obtained from official and maintained sources. 
Additionally, it could be misleading for individuals who don't have the Telegraf code but only the binary itself.

Reply:
Okay, what about a link to that file in the official repo then?

Also, if they only have the telegraf binary, they also wouldn't have this README file.

Reply:
@Hipska 

> Okay, what about a link to that file in the official repo then?

Each CPU model has its dedicated event file on the mentioned [perfmon](https://github.com/intel/perfmon) repo. This README is probably not the place where we want to create a list of links to CPU models in a separate repo (and no one can guarantee that paths or file names won't change within that repo).



> Also, if they only have the telegraf binary, they also wouldn't have this README file.

That's true, but users can be redirected to this README from the Telegraf documentation (such as the Plugin Directory: https://docs.influxdata.com/telegraf/v1/plugins/#input-intel_powerstat).

Reply:
Yeah, so then the link would work, right? But it's okay for me, this convo can be marked as resolved.